foreman_ansible.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
             ['package.json']
   s.test_files = Dir['test/**/*']
 
-  s.add_dependency 'acts_as_list', '~> 1.0.3'
+  s.add_dependency 'acts_as_list', '~> 1.2'
```

Comment:
Technically we could also do `~> 1.0, >= 1.0.3` if you prefer.

Reply:
Do you see any valid reason for using the wider range?

Reply:
The only I can think of is packaging.
Today we package 1.0.3, so setting 1.2 here would mean we need immediately also update packaging. (Which we can do! Just spelling it out loud here)

Reply:
Thanks Evgeni, so I'm good with merging it as is.